### PR TITLE
[Merged by Bors] - feat(Condensed): generalise universes in `TopComparison`

### PR DESCRIPTION
**Pull request by dagurtomas**
---

[Open in Gitpod](https://gitpod.io/from-referrer/)

### CODE REVIEW

**Review by grunweg — APPROVED** [review]
Seems straightforward.

File: Mathlib/Condensed/TopComparison.lean [hunk 1]
```diff
@@ -30,8 +30,8 @@ universe w w' v u
 
 open CategoryTheory Opposite Limits regularCoverage ContinuousMap
 
-variable {C : Type u} [Category.{v} C] (G : C ⥤ TopCat.{v})
```
Comment (eric-wieser) [inline Mathlib/Condensed/TopComparison.lean:33]: I don't think this is necessarily more general; omitting `.{}` means that a universe metavariable is used instead, which could potentially be populated with something *less* general. Could you keep the explicit universe names here (perhaps using more of them) to ensure the generality is what you intend?

Comment (dagurtomas) [reply Mathlib/Condensed/TopComparison.lean:33]: In this case it is completely general. I've added explicit names for the universes.

**Review by eric-wieser — APPROVED** [review]
bors merge

Thanks!

**Pull request closed without merging**